specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,544 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-08-31-preview",
+    "title": "ManagedServiceIdentityClient",
+    "description": "The Managed Service Identity Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+   ],
+   "consumes": [
+    "application/json"
+   ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.ManagedIdentity/operations": {
+      "get": {
+        "operationId": "Operations_List",
+        "description": "Lists available operations for the Microsoft.ManagedIdentity provider",
+        "x-ms-examples": {
+          "MsiOperationsList": {
+            "$ref": "./examples/MsiOperationsList.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the list of available operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ManagedIdentity/userAssignedIdentities": {
```

Comment:
From discussion with ManoharP, it seems like the plan was to use "identities" as the resource type name. And then the "systemAssignedIdentities" is the other resource type which you declare in the manifest and this is not visible to the end user.

Reply:
We had a discussion with the PM team. The decision was to keep the current naming scheme, which is:
userAssignedIdentities = explicit identity

Reply:
What is the resourceIdentityType enum value used when associating an explicit identity with a resource? Presumably that enum value should be userAssigned? 
 i

Reply:
When associating an explicit identity with a resource, the enum value that we use is “userAssigned”.